vendor/github.com/hashicorp/terraform/helper/schema/data_source_resource_shim.go

```diff
@@ -32,7 +32,7 @@ func DataSourceResourceShim(name string, dataSource *Resource) *Resource {
 
 	// FIXME: Link to some further docs either on the website or in the
 	// changelog, once such a thing exists.
-	dataSource.deprecationMessage = fmt.Sprintf(
+	dataSource.DeprecationMessage = fmt.Sprintf(
```

Comment:
It's odd that there are updates to the vendor files without a changed commit hash in `vendor/vendor.json`. Were these made directly? If so they'll get wiped out when `govendor` is next used to update dependencies. Would it be possible to get these changes from different versions of these libraries instead? Here's the [Govendor Cheat Sheet](https://github.com/kardianos/govendor/wiki/Govendor-CheatSheet) if it helps.

Reply:
@tyrannosaurus-becks thanks for the helpful reply

I was not sure which dependency solution you are using - so I adjusted manually what I needed from the terraform HEAD

Do you think it is ok to update the whole terraform dependencies to a newer version?
That make probably other adjustments necessary?

I can try that if I have some time...

Reply:
Thanks for offering to do that! I'd love to leave Terraform's dependencies as unruffled as possible and shoot for the minimum change on each PR. Ah yes, it's not in the docs, but we have been using [GoVendor](https://github.com/kardianos/govendor/wiki/Govendor-CheatSheet) for this repo. Is that `DeprecationMessage` exported on later versions of Terraform? Do you happen to know what the first release was where it was exported?

Reply:
@tyrannosaurus-becks sorry for the delay with this PR

I tried today to come to a more current version of vendored terraform - created a new PR for it https://github.com/terraform-providers/terraform-provider-vault/pull/341

Some input how to proceed is very much appreciated!
Thanks